karma.conf.js

```diff
@@ -51,7 +51,7 @@ module.exports = function (config) {
      */
     files: [{
       pattern: './spec-bundle.js',
-      watched: false
+      watched: false,
```

Comment:
I often omit the trailing comma for the last element.

What's the reason behind this change?